README.md

```diff
@@ -33,8 +29,7 @@ The samples were developed and tested using [Mars 4.5.0](https://eclipse.org/mar
 
 ## Using remote input PDF files with the samples
 
-By default, the samples run with pre-packaged input PDF files that are treated as resources of the samples. However, you can use your own input PDF files 
-including those that are stored remotely on a shared network with the samples. As long as the shared network where the PDF files reside is mounted on your local machine it's treated a part of your local file system.
+By default, the samples run with pre-packaged input PDF files that are treated as resources of the samples. However, you can use your own input PDF files, including those that are stored remotely on a shared network with the samples. As long as the shared network where the PDF files reside is mounted on your local machine it's treated a part of your local file system.
```

Comment:
"A paragraph is simply one or more consecutive lines of text, separated by one or more blank lines. (A blank line is any line that looks like a blank line — a line containing nothing but spaces or tabs is considered blank.) Normal paragraphs should not be indented with spaces or tabs."—[Markdown syntax documentation](http://daringfireball.net/projects/markdown/syntax#p).

Some Markdown viewers don't do as good a job as others, but since this file is usually for viewing in GitHub, you can look at the version of the file in the pull request with the **View** button to see how GitHub formats it.